local/bin/4-load-initial-policy

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
I would rename this `load-secrets` since that's the important part of what's happening. We could also move it back to happen immediately after the Conjur deployment so that the demo sort of covers Conjur-specific stuff and then PCF-specific stuff without having to swap back and forth. Might make it easier on the clients.

Reply:
I renamed it into `4-load-secrets-into-conjur` and moved it.